src/views/AddTimerView.tsx

```diff
@@ -0,0 +1,153 @@
+                                    totalRounds: +e.target.value,
+                                }))
+                            }
+                        />
+                    </>
+                );
+            case 'tabata':
+                return (
+                    <>
+                        <input
+                            type="number"
+                            placeholder="Work Duration (seconds)"
+                            value={config.workSeconds || ''}
+                            onChange={e =>
+                                setConfig((prev: any) => ({
+                                    ...prev,
+                                    workSeconds: +e.target.value,
+                                }))
+                            }
+                        />
+                        <input
+                            type="number"
+                            placeholder="Rest Duration (seconds)"
+                            value={config.restSeconds || ''}
+                            onChange={e =>
+                                setConfig((prev: any) => ({
+                                    ...prev,
+                                    restSeconds: +e.target.value,
+                                }))
+                            }
+                        />
+                        <input
+                            type="number"
+                            placeholder="Total Rounds"
+                            value={config.totalRounds || ''}
+                            onChange={e =>
+                                setConfig((prev: any) => ({
+                                    ...prev,
+                                    totalRounds: +e.target.value,
+                                }))
+                            }
+                        />
+                    </>
+                );
+            case 'stopwatch':
+                return <input type="number" placeholder="Total Time (seconds)" value={config.totalSeconds || ''} onChange={e => setConfig({ totalSeconds: +e.target.value })} />;
+            default:
+                return null;
+        }
+    };
+
+    return (
+        <div className="add-timer-container">
+            <h2 className="add-timer-heading">Add Timer</h2>
+            {error && <div className="error-message">{error}</div>}
+
+            {/* Timer type options */}
+            <div className="timer-type-options">
+                {(['countdown', 'xy', 'tabata', 'stopwatch'] as const).map(option => (
+                    <button
+                        key={option}
+                        className={`timer-option-button ${type === option ? 'active' : ''}`}
+                        onClick={() => {
+                            setType(option);
+                            setConfig({});
+                            setError(null);
+                        }}
+                    >
+                        {option.charAt(0).toUpperCase() + option.slice(1)}
```

Comment:
```
{option.charAt(0).toUpperCase() + option.slice(1)}
```
It's hard to understand what this code is doing. This is a good opportunity to hide this logic behind a well-named helper function that explains what you're doing.